packages/webapp/src/components/Finances/Transaction/ExpandedContent/LabourTable.jsx

```diff
@@ -0,0 +1,182 @@
+    [transaction],
+  );
+
+  const taskTotalHours = useMemo(
+    () => calculateTotalHours(transaction.items.TASK_TYPE),
+    [transaction],
+  );
+
+  const employeeColumns = useMemo(
+    () => [
+      {
+        id: 'employee',
+        label: t('SALE.LABOUR.TABLE.EMPLOYEE'),
+        Footer: <div className={styles.footer}>{t('SALE.LABOUR.TABLE.DAILY_TOTAL')}</div>,
+        columnProps: {
+          style: { width: '33.33%' },
+        },
+      },
+      {
+        id: 'time',
+        label: t('SALE.LABOUR.TABLE.TIME'),
+        format: (d) => `${d.time} h`,
+        align: 'right',
+        Footer: <b>{employeeTotalHours} h</b>,
+        columnProps: {
+          style: { width: '33.33%' },
+        },
+      },
+      {
+        id: 'labourCost',
+        label: t('SALE.LABOUR.TABLE.LABOUR_COST'),
+        format: (d) => {
+          return (
+            <span>
+              {currencySymbol}
+              {Math.abs(d.labourCost).toFixed(2)}
+            </span>
+          );
+        },
+        align: 'right',
+        Footer: (
+          <b>
+            {currencySymbol}
+            {Math.abs(transaction.amount).toFixed(2)}
+          </b>
+        ),
+        columnProps: {
+          style: { width: '33.33%' },
+        },
+      },
+    ],
+    [transaction, currencySymbol],
+  );
+
+  const tasksColumns = useMemo(
+    () => [
+      {
+        id: 'taskType',
+        label: t('SALE.LABOUR.TABLE.TASK'),
+        Footer: <div className={styles.footer}>{t('SALE.LABOUR.TABLE.DAILY_TOTAL')}</div>,
+        component: 'th',
+        columnProps: {
+          style: { width: '33.33%' },
+        },
+      },
+      {
+        id: 'time',
+        label: t('SALE.LABOUR.TABLE.TIME'),
+        format: (d) => `${d.time} h`,
+        align: 'right',
+        Footer: <b>{taskTotalHours} h</b>,
+        columnProps: {
+          style: { width: '33.33%' },
+        },
+      },
+      {
+        id: 'labourCost',
+        label: t('SALE.LABOUR.TABLE.LABOUR_COST'),
+        format: (d) => {
+          return (
+            <span>
+              {currencySymbol}
+              {Math.abs(d.labourCost).toFixed(2)}
+            </span>
+          );
+        },
+        align: 'right',
+        Footer: (
+          <b>
+            {currencySymbol}
+            {Math.abs(transaction.amount).toFixed(2)}
+          </b>
+        ),
+        columnProps: {
+          style: { width: '33.33%' },
+        },
+      },
+    ],
+    [transaction, currencySymbol],
+  );
+
+  return (
+    <div className={styles.labourTable}>
+      <StateTab
+        className={styles.tabs}
+        state={activeTab}
+        setState={setActiveTab}
+        tabs={[
+          {
+            label: t('SALE.LABOUR.EMPLOYEES'),
+            key: 'employee',
+          },
+          {
+            label: t('SALE.LABOUR.TASKS'),
+            key: 'tasks',
+          },
+        ]}
+      />
+      {activeTab === 'employee' && (
```

Comment:
NIT, but I'd make an enum with the options for `activeTab`

Reply:
I'm not 100% sure I did that right; can you take a look and make sure it fits what you were thinking?

Reply:
Yup looks good! 🙂